compass/ocean/mesh/remap_topography.py

```diff
@@ -138,11 +142,28 @@ def run(self):
             out_var = rename[option]
             ds_out[out_var] = ds_in[in_var]
 
+        ds_out['landIceFloatingFracObserved'] = \
+            ds_out.landIceFracObserved - ds_out.landIceGroundedFracObserved
+
+        # make sure fractions don't exceed 1
+        for var in ['landIceFracObserved', 'landIceGroundedFracObserved',
+                    'landIceFloatingFracObserved', 'oceanFracObserved']:
+            ds_out[var] = np.minimum(ds_out[var], 1.)
```

Comment:
Do we also need a check that `oceanFracObserved + landIceFracObserved == 1` (or 1 + eps)? I haven't checked.

Reply:
There is a third category: bare bedrock.  This will be culled out as land later on but it is present at this stage.  So it won't be possible to sanity check that unless we want to also add the 3rd category as a mask. I'd slightly prefer not to add that because sanity checking 3 masks is more complex.  But we could consider it.

Reply:
Let's leave it as-is then. thanks!